### ISSUE TITLE
Pull system logs from machines

### ISSUE THREAD

**Issue by giovannt0** [google/cloud-forensics-utils#103]
Explore possibilities of retrieving system logs (e.g. /var/log/syslog) and other comon logs directly from libcloudforensics

**Comment by hacktobeer** [comment]
We already support this for GCP -> VM syslogs are logged in Google Cloud logs.

Are you thinking about interfacing with the filesystem of cloud disks and pulling specific (log) files?

**Comment by giovannt0** [comment]
Yes, exactly. I aslo don't know what AWS currently offers for these kind of logs but it could be nice to have a method `get_log(SYSLOG)` that just fetches the file

**Comment by hiddenillusion** [comment]
For AWS, you're best option is a standard remote transfer (e.g. SCP) unless you have some agent installed.

Some benefits will come out of using SSM agent on top of this, but also adds more requirements.
